server/routes/termdb.singlecellSamples.ts

```diff
@@ -51,6 +51,7 @@ function init({ genomes }) {
 			if (!ds) throw 'invalid dataset name'
 			if (!ds.queries?.singleCell) throw 'no singlecell data on this dataset'
 			result = (await ds.queries.singleCell.samples.get(q)) as TermdbSinglecellsamplesResponse
+			if (!('sameLegend' in result)) result.sameLegend = false
```

Comment:
how about change this line to:
result.sameLegend = ds.queries.singleCell.samples.sameLegend
thus no need to change mds3.gdc.js